src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -383,7 +383,7 @@ record = _serverStore.Cluster.ReadDatabase(context, Name, out index);
                             _logger.Info("An unhandled exception closed the cluster transaction task", e);
                         }
                     }
-                }, null, ThreadNames.ForClusterTransactions("Cluster Transaction", Name));
+                }, null, ThreadNames.ForClusterTransactions($"Cluster Transaction {Name}", Name));
```

Comment:
Use the same instance that you you use above

Reply:
done